cpp/src/arrow/flight/sql/odbc/flight_sql/ui/dsn_configuration_window.cc

```diff
@@ -280,7 +280,7 @@ int DsnConfigurationWindow::CreateEncryptionSettingsGroup(int posX, int posY, in
       rightCheckPosX, rowPos - 2, 20, 2 * ROW_HEIGHT, "",
       ChildId::DISABLE_CERT_VERIFICATION_CHECKBOX, disableCertVerification);
 
-  rowPos += INTERVAL + static_cast<int>(1.5 * ROW_HEIGHT);
+  rowPos += INTERVAL + static_cast<int>(1.5 * static_cast<int>(ROW_HEIGHT));
```

Comment:
This change is duplicated at https://github.com/apache/arrow/pull/47517. But it is required to fix the build. After this PR is merged, I will remove duplication at https://github.com/apache/arrow/pull/47517